TF-3754 add .spec to basics match-regex

joecan opened the pull request:
##### *To:*
@kevinzang @udemy/team-f @inancsevinc

##### *What:*
In order to have spec files in the webpack directory we need to update filnames/match-regex to allow *.spec.js and move jasmine-addons from udemy-website to udemy-basic.

##### *JIRA:*
https://udemyjira.atlassian.net/browse/TF-3754

##### *What did you test:*
yarn test

##### *What dashboards will you be monitoring:*
N/A

joecan commented:
Should have I just done this in `static/webpack/.eslintrc.js` instead?

kevinzang commented:
@joecan good idea!

Let's keep the change to eslint-config-udemy-basics/parts/filenames.js and do the rest in static/webpack/.eslintrc.js:

```
module.exports = {
    extends: ['udemy-basics', 'udemy-jasmine-addons'],
};
```

joecan committed:
[TF-3754] add .spec to basics match-regex

kevinzang reviewed (approved):
@joecan LGTM!

Commit:
Publish

 - babel-polyfill-udemy-website@9.0.6
 - babel-preset-udemy-website@11.0.3
 - eslint-config-udemy-basics@8.0.2
 - eslint-config-udemy-website@12.0.5
